### PR TITLE
Remove old twitter spinner ( obsolete - follow up from #179 )

### PR DESCRIPTION
**Pull request by ligi**

### CODE REVIEW

**Comment by cbalster** [comment]
you missed the errormessage layout

**Comment by ligi** [comment]
yepp - just seen it

**Comment by ligi** [comment]
Updated

File: app/src/main/res/layout/fragment_twitter.xml [hunk 1]
```diff
@@ -53,7 +45,7 @@
         android:id="@+id/twitter_error"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_below="@id/progressSpinner"
+        android:layout_centerInParent="true"
```
Comment (cbalster) [inline app/src/main/res/layout/fragment_twitter.xml]: Either centerInParent or centerHorizontal + centerVertical is needed, right?

**Pull request merged**